rtf_vis_tool/src/Components/gtsfm_node_list.js

```diff
@@ -0,0 +1,53 @@
+//List of all 43 Nodes in GTSFM Visualization Graph
+//imported in DivGraph.js
+
+const gtsfm_node_set = ['absolute Rs',
```

Comment:
can we list these alphabetically, or in some topological order-based grouping for now?

Reply:
yup, I'll order it alphabetically.